src/UI/UIView.cpp

```diff
@@ -1348,6 +1806,64 @@ void UIView::addTransferFunction()
   }
   m_tfns.push_back(trfntc);
 }
+
+void UIView::get_Quantiles(int row)
+{
+  m_copy_trnfnct_name = m_tfns[row].Name;
+  float q_min = 0;
+  float q_max = 0;
+  tfn_widget[m_trnfnc_table_selection].get_Quantiles(q_min, q_max);
+  m_histogram_quantiles[0] = q_min;
+  m_histogram_quantiles[1] = q_max;
+  m_trn_fct_options_window = true;
+}
+
+void UIView::set_volume_time_info(time_t time)
+{
+
+  tm* time_info = localtime(&time);
+  if (time_info)
+  {
+    bool pm = time_info->tm_hour >= 12;
```

Comment:
is there a builtin/simple package that handles time formatting?

Reply:
Probably. Time management on desktops apps depend on the OS. I'll have to check for multi platform light libraries that might help. C++ boost might have one, but that library is too heavy.